packages/orm/src/BaseEntity.ts

```diff
@@ -20,6 +20,7 @@ import {
 export abstract class BaseEntity implements Entity {
   abstract id: string | undefined;
   private __isNewEntity: boolean = true;
+  private __isTouched: boolean = false;
```

Comment:
Maybe squirrel this into `__orm`? Maybe the `__orm` thing wasn't a good idea, we look like 3/4rd inside `__orm`, a few things sneaking out...